pkg/arguments/normalize.go

```diff
@@ -25,6 +27,8 @@ func NormalizeFlags(_ *pflag.FlagSet, name string) pflag.NormalizedName {
 		name = NewWorkerIAMRole
 	case DeprecatedControlPlaneIAMRole:
 		name = NewControlPlaneIAMRole
+	case DeprecatedEnvFlag:
```

Comment:
@tirthct Can you please update the unit tests for the `NormalizeFlags` function to account for your changes. Thanks!

Reply:
Updated